go.mod

```diff
@@ -1,66 +1,70 @@
 module github.com/CiscoDevNet/terraform-provider-aci/v2
 
-go 1.18
+go 1.21
```

Comment:
did you need the upgrade of mod? should we do this in separate PR already created PR for this if required?

Reply:
ok